uint8 robot_id
uint8 wheel_right
uint8 wheel_left
